docs/kb/integrations/wiz.md

```diff
@@ -0,0 +1,90 @@
+---
+slug: wiz
+append_help_link: true
+title: View Semgrep findings in Wiz's Security Graph
+hide_title: true
+description: "Learn how to view Semgrep findings in Wiz's Security Graph."
+tags:
+ - Wiz
+ - Semgrep AppSec Platform
+---
+
+# View Semgrep findings in Wiz's Security Graph
+
+Semgrep integrates with Wiz by calling Wiz’s GraphQL API endpoints and uploading your static analysis (SAST) vulnerability findings to a dedicated Amazon S3 bucket. Your Semgrep SAST vulnerability findings are mapped to the same correlated repository scanned by Wiz and enriched by any available inventory and runtime-related data, such as clusters, pods, containers, cloud configurations, and more. Semgrep's goal is to give you a holistic view of your code and infrastructure security so that you can focus on what matters most.
```

Comment:
Some stuff here is out-of-date and the other is a little difficult to read. How does this sound?

```
Semgrep integrates with Wiz by establishing a secure connection with Wiz’s API endpoints. If your Wiz instance has a security graph enrichment integration, it is possible to view SAST vulnerabilities that were identified by Semgrep in the correlated repos in your CNAPP platform. Semgrep's goal is to give you a holistic view of your code and infrastructure security so that you can focus on what matters most. For Wiz users with a [Code Security](https://www.wiz.io/solutions/wiz-for-code-security) license, this integration takes effect automatically when you create a Wiz Cloud Insights account. 
```